implementations/rust/ockam/ockam_command/tests/bats/load/orchestrator.bash

```diff
@@ -14,6 +14,13 @@ function skip_if_orchestrator_tests_not_enabled() {
   fi
 }
 
+function skip_if_docs_tests_not_enabled() {
```

Comment:
Could you please add a section in the README to explain how to run those tests?
One interesting part is the setup. Do they all work in a nix shell? That would be fantastic.

Reply:
We use Nix for most of the setup, I had to use the [supported Docker image of Influx](https://docs.influxdata.com/influxdb/v2/install/?t=Docker#persist-data-outside-the-influxdb-container) as Nix Influxd wasn't working accurately. I added how to run the Docs in the readme section.